zerver/tests/test_signup.py

```diff
@@ -868,17 +868,18 @@ def test_register(self) -> None:
         with queries_captured() as queries, cache_tries_captured() as cache_tries:
             self.register(self.nonreg_email("test"), "test")
         # Ensure the number of queries we make is not O(streams)
-        self.assert_length(queries, 90)
+        self.assert_length(queries, 96)
```

Comment:
The six queries here are - one each for getting members and full members system group, one each for creating `UserGroupMembership` objects for members and new members group and one each for `active_user_ids(user_group.realm_id)` in the `do_send_user_group_members_update_event` for adding user to member and full members groups.